### ISSUE TITLE
Backup set showing as 0

### ISSUE THREAD

**Issue by nrahuf** [jlesage/docker-crashplan-pro#181]
Received my regular email today - and it says that nothing is backed up in my crash plan. - I had 1.2TB backed up that is now showing as 0 - I haven't deleted files or changed the files that were backing up.

did log in to crash plan and immediately got an upgrade failed error. 

also deleted and reinstalled the container but still as showing as 0 

what happened?

**Comment by nrahuf** [comment]
when I check my folders in crashplan app in the container - they all seem empty now although on my NAS they have files in the folders

**Comment by gatorheel** [comment]
You need to update your container.  In-app Auto-update doesn't work for containers.

https://github.com/jlesage/docker-crashplan-pro/releases/tag/v2.7.1

**Comment by nrahuf** [comment]
I believe I am running the latest - I updated a few days ago. will try again and see if that fixes it - this wouldn't show my folders being empty would it?

**Comment by nrahuf** [comment]
I have reinstalled with 2.71 - which I believe I was running before and still showing all folders as empty!

**Comment by nrahuf** [comment]
for some reason all my folders are showing as empty - nothing in the config has changed.

**Comment by jlesage** [comment]
Maybe the permissions of your files changed?

**Comment by nrahuf** [comment]
so I cannot see any of the folders just gone into manage files and cannot see any of my folder structure - so not sure what has changed - I haven't changed anything on my NAS 

Anyone to open to a team view session to help me fix this as I think its beyond my knowledge scope.

**Comment by jlesage** [comment]
If you login into the container via `docker exec -ti <container name> sh`, do you see your files under `/storage`?

**Comment by gatorheel** [comment]
Can you also share the command you're using to (re)create your container?

**Comment by nrahuf** [comment]
ok when I do the upgrade from docker in qnap this where the issue occurs if I set up a new container I can see the folder structure - 

if I reinstall the container using 

docker run [-d] \
    --name=crashplan-pro \
    [-e <VARIABLE_NAME>=<VALUE>]... \
    [-v <HOST_DIR>:<CONTAINER_DIR>[:PERMISSIONS]]... \
    [-p <HOST_PORT>:<CONTAINER_PORT>]... \
    jlesage/crashplan-pro

the folder structure exists - when I do this it installs an old version not the latest so I keep getting asked to upgrade.

**Comment by nrahuf** [comment]
not sure what has happened - I just started from scratch - folder structure seems to have to have come back - think it may be the docker images being pulled in the snap container station - this time I used the command line to do the upgrade after installing from scratch

**Comment by caseylucas** [comment]
I'm not sure your command to "reinstall the container" will actually get you a new container.  For reference, here is the scripts i use on my qnap:
- upgrade https://gist.github.com/caseylucas/4cf835ad6a31bbfa7ed43c7b69118bf4
- start https://gist.github.com/caseylucas/e9bc9ab7716fae17d23800edd9a0b986

**Comment by nrahuf** [comment]
thanks Casey - I have started from scratch - seems to have found the folder structure again - and is currently coopering back ups.

**Comment by nrahuf** [comment]
ok not sure what is going on but the app seems to only find 732Gb whereas I had 1.2TB backed up before - could this be with the file changes I know circa 450GB is time machine backups - assume these are no longer being backed up by CrashPlan now? that correct? any idea of a way around it?

**Comment by jlesage** [comment]
Yes this is possible.  You can look at https://support.code42.com/CrashPlan/6/Troubleshooting/What_is_not_backing_up to see if your time machine backups fall into the new exclusion list.  Also, in CrashPlan, you can click on the "Restore Files" button to navigate through what has been backup.
